app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt

```diff
@@ -44,4 +48,23 @@ class CovidWatchApplication : Application() {
             downloadRequest
         )
     }
+
+    fun executePublicContactEventsRefresh() {
+        val downloadRequest = OneTimeWorkRequestBuilder<ContactEventsDownloadWorker>()
+            .setConstraints(getRefreshConstraints())
+            .build()
+
+        WorkManager.getInstance(this).enqueueUniqueWork(
+            ContactEventsDownloadWorker.WORKER_NAME,
```

Comment:
We may have to provide different `uniqueWorkName` for this `OneTimeWorkRequestBuilder`, because otherwise it will probably `ExistingWorkPolicy.REPLACE` our `PeriodicWorkRequestBuilder`

Reply:
Or maybe try to use `ExistingWorkPolicy.KEEP`? (needs tests to be tested)

Reply:
Good point! Will test it and report what happens.

Reply:
Even though the documentation hints that both requests are separate, it looks like they are not so I gave them different names.